test/test_ops.py

```diff
@@ -60,23 +61,29 @@ def _autograd_grad(outputs, inputs, grad_outputs=None, retain_graph=False, creat
     return tree_unflatten(result, inputs_spec)
 
 
-def diff_arg(arg):
+def diff_arg(arg, is_jvp=False):
```

Comment:
nit: flags are hard to read. if `is_jvp=True` then we know we're dealing with jvp, if `is_jvp=False` then what APIs are we testing?

This is probably better as
```
def is_floating_point(tensor):
  ...

def tensor_requires_grad(tensor):
  return tensor.requires_grad

def diff_arg(arg, is_differentiable_fn=tensor_requires_grad):
  ...
```

Reply:
Yeah `is_jvp` is probably too jvp-centric, but I still think this argument is more of a binary thing (what would be a third function that one would pass here?). Perhaps a more "neutral" flag would be better. I'm going to rename to `requires_grad` instead for now.

Reply:
A more neutral flag sounds good to me